### PR TITLE
Add p2 hub integrations sidebar menu item

### PR DESCRIPTION
**Pull request by unDemian**
#### Changes proposed in this Pull Request:

This PR adds Tools \ Integrations sidebar menu item for p2 hub sites, already tested in D65214-code

Markup on 2022-06-07 at 5:43:47 PM

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
- Apply D82137-code 
- Changes to Calypso sidebar menu.
- For P2 spaces -> Tools \ Marketing and Tools \ Earn should not be visible
- For P2 Hubs -> Tools \ Integrations should be visible but only for Administrators

### CODE REVIEW

File: projects/plugins/jetpack/modules/masterbar/admin-menu/class-p2-admin-menu.php [hunk 1]
```diff
@@ -139,7 +143,11 @@ private function remove_menus_for_hub() {
 		// Hide appearance.
 		remove_menu_page( $this->appearance_slug );
 		// Tools.
-		remove_menu_page( $this->tools_slug );
+		remove_submenu_page( $this->tools_slug, 'https://wordpress.com/marketing/tools/' . $this->domain );
+		remove_submenu_page( $this->tools_slug, 'https://wordpress.com/earn/' . $this->domain );
+		remove_submenu_page( $this->tools_slug, 'https://wordpress.com/import/' . $this->domain );
+		remove_submenu_page( $this->tools_slug, 'https://wordpress.com/export/' . $this->domain );
+		add_submenu_page( $this->tools_slug, esc_attr__( 'Integrations', 'jetpack' ), __( 'Integrations', 'jetpack' ), 'manage_options', 'https://wordpress.com/marketing/connections/' . $this->domain, null, 0 );
```
Comment (samiff) [inline projects/plugins/jetpack/modules/masterbar/admin-menu/class-p2-admin-menu.php:150]:
```suggestion
		add_submenu_page( $this->tools_slug, __( 'Integrations', 'jetpack' ), __( 'Integrations', 'jetpack' ), 'manage_options', 'https://wordpress.com/marketing/connections/' . $this->domain, null, 0 );
```

Is there a reason behind using `esc_attr__` vs just `__` here?

**Review by lezama — APPROVED** [review]
:shipit:

**Comment by lezama** [comment]
works as described

**Review by lezama — APPROVED** [review]

**Review by lezama — APPROVED** [review]

**Review by oskosk — APPROVED** [review]
IPF

**Comment by oskosk** [comment]
Merged it so if something breaks, it's my fault

**Pull request merged**